Software/README.md

```diff
@@ -987,155 +987,192 @@ Where error_staus value can be NO_ERROR in case data are exchanged with no commu
 **Note:** All user scripts that import the pijuice module must now be run as Python3
 
 #### PiJuice Status
+
 **GetStatus()**
 
 Gets basic PiJuice status information about power inputs, battery and events.
 Returns:
-'data':{
+```python
+{'data':{
```

Comment:
Made some small improvement to some of the returns - added the wrapping curly brackets.